librz/debug/p/native/linux/linux_debug.c

```diff
@@ -872,11 +872,59 @@ RzList /*<RzDebugPid *>*/ *linux_pid_list(int pid, RzList /*<RzDebugPid *>*/ *li
 	return list;
 }
 
+static int thread_find(int *tid, RzDebugPid *t, void *user) {
+	return (t && (t->pid == *tid)) ? 0 : 1;
+}
+
+RZ_API RZ_OWN RzDebugPid /*<RzDebugPid *>*/ *rz_debug_get_thread(RzList *th_list, int tid) {
```

Comment:
The type annotation is entirely unnecessary here, it's only for complex types, e.g. `RzList`, `RzPVector`, etc

Reply:
i think `rz-bindings` needs to know what `RzList *th_list` is.

Reply:
But i agree that `/*<RzDebugPid *>*/ ` is not needed since is not a list or vector or pvector.